### DISCUSSION TITLE
Laser point detection

### DISCUSSION THREAD

**Discussion by katzlea** [biigle/core#463] [Q&A] [answered]
Hi,

I keep getting this error : "Delphi laser point detection can't be performed. Images must have equal count of manually annotated laser points."
It's probably because not all of my images have laser points. Is it possible to manually annotate laser points on my images and have an idea of measurements in my annotations reports? 
Not all of my images have them, but I really to wish to have the measurement info on the ones that do!

Thanks for reading!

**Comment by mzur** [comment, marked as answer]
The error message means that there are images with laser point (LP) annotations but some images have a different count of LP annotations than others (e.g. one image has 3 LP annotations and another has 2). The automatic LP detection algorithm requires an equal count of LP annotations for each image.

You can either annotate a few images manually and then run the automatic LP detection or you can annotate all images manually. Manual annotations can be more accurate (and the measurement info will be, too) but the automatic method may be faster.

**Reply by katzlea** [reply]
Hi @mzur,
Thanks for responding so quickly. I thought that it didn't work because some had 0... But i check again and it turned out that I forgot to annotate one point in a picture. Now it works! Thanks a lot for your help!
